src/index.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './pages/App';
 import * as serviceWorker from './actions/serviceWorker';
+import './styles/App.scss';
```

Comment:
This is already imported in App.tsx, so if it's better to move here so the app isn't prematurely rendered without the proper global styles, make sure you delete it there first.

Reply:
Removed from index.tsx